pennylane/math/single_dispatch.py

```diff
@@ -46,6 +46,7 @@ def _i(name):
 # the following is required to ensure that SciPy sparse Hamiltonians passed to
 # qml.SparseHamiltonian are not automatically 'unwrapped' to dense NumPy arrays.
 ar.register_function("scipy", "to_numpy", lambda x: x)
+ar.register_function("scipy", "shape", np.shape)

```

Comment:
this almost feels like a bug fix? was dispatching to scipy causing warnings to be raised on the scipy end? or was it something else?

Reply:
It was a deprecation warning.
```
ops/qubit/test_sparse.py::TestSparse::test_label
  /Users/christina/Prog/pl/lib/python3.8/site-packages/autoray/autoray.py:84: DeprecationWarning: scipy.shape is deprecated and will be removed in SciPy 2.0.0, use numpy.shape instead
    return get_lib_fn(backend, fn)(*args, **kwargs)
```